diff --git a/Alc/ALc.c b/Alc/ALc.c
index f7f7b63..0aac31e 100644
--- a/Alc/ALc.c
+++ b/Alc/ALc.c
@@ -798,9 +798,6 @@ BOOL APIENTRY DllMain(HINSTANCE hModule, DWORD reason, LPVOID lpReserved)
     switch(reason)
     {
         case DLL_PROCESS_ATTACH:
-            /* Pin the DLL so we won't get unloaded until the process terminates */
-            GetModuleHandleExW(GET_MODULE_HANDLE_EX_FLAG_PIN | GET_MODULE_HANDLE_EX_FLAG_FROM_ADDRESS,
-                               (WCHAR*)hModule, &hModule);
             alc_init();
             break;
 
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0087edb..b98bdbb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -40,7 +40,7 @@ OPTION(ALSOFT_HRTF_DEFS "Install HRTF definition files" ON)
 
 
 IF(WIN32)
-    SET(LIBNAME OpenAL32)
+    OPTION(LIBNAME "Library name" OpenAL32)
     ADD_DEFINITIONS("-D_WIN32 -D_WIN32_WINNT=0x0502")
 
     # This option is mainly for static linking OpenAL Soft into another project
